app/templates/response/add_closing.html

```diff
@@ -56,6 +56,9 @@
 
             <div class="third" hidden>
                 <h4 id="edit-body-header">Edit Email Body</h4>
+                <div id="additional-details-warning" class="alert alert-warning hidden" role="alert">
+                    <strong>Please provide additional details when selecting this reason.</strong>
```

Comment:
Future improvement - we could pull this from the from the database instead of hardcode?